### ISSUE TITLE
✅ 🔗 🐞 New Tabs Widget excruciatingly slow  (Duplicate of #24076)

### ISSUE THREAD

**Issue by tamelander** [elementor/elementor#27417]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

The new Tabs widget is painfully slow. I add a tab, or duplicate an existing tab, and then try to save the page. The editor "thinks" for many minutes before the save progress bar finishes. If someone in charge here want to take a look, I can create a staging site for debugging and create an admin user. 

### Steps to reproduce

1. Add new tab to existing tabs widget.
2. Try to save.
3. Wait five minutes.

### Expected behavior

1. Add new tab to existing tabs widget.
2. Try to save.
3. Page saves.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.17
	PHP Version: 8.2.18
	PHP Memory Limit: 2048M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 2048M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.3
	Site URL: redacted
	Home URL: redacted
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 2048M
	Max Memory limit: 2048M
	Permalink Structure: /%postname%/
	Language: sv_SE
	Timezone: Europe/Stockholm
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.6.14
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: sv-SE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Sidebar Menu
		Version: 9.5.2
		Author: OnPoint Plugins

	Asset CleanUp: Page Speed Booster
		Version: 1.3.9.3
		Author: Gabe Livan

	Astra Pro
		Version: 4.6.9
		Author: Brainstorm Force

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Classic Widgets
		Version: 0.3
		Author: WordPress Contributors

	Connect Polylang for Elementor
		Version: 2.4.4
		Author: Creame

	Content Views
		Version: 3.7.2
		Author: Content Views

	Duplicate Menu
		Version: 0.2.2
		Author: Jon Christopher

	Elementor
		Version: 3.21.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.21.3
		Author: Elementor.com

	Feed Them Social - Page, Post, Video and Photo Galleries
		Version: 4.2.7
		Author: SlickRemix

	Formidable Forms
		Version: 6.9.1
		Author: Strategy11 Form Builder Team

	Formidable Forms Pro
		Version: 6.9.2
		Author: Strategy11

	Formidable Views
		Version: 4.0.05
		Author: Strategy11

	IndexNow
		Version: 2.6.5
		Author: Collins Agbonghama

	Inline Google Spreadsheet Viewer
		Version: 0.13.2
		Author: Meitar Moscovitz <meitarm+wordpress@gmail.com>

	LiteSpeed Cache
		Version: 6.2.0.1
		Author: LiteSpeed Technologies

	Max Mega Menu
		Version: 3.3.1
		Author: megamenu.com

	Max Mega Menu - Pro Addon
		Version: 2.4
		Author: megamenu.com

	OMGF
		Version: 5.9.0
		Author: Daan from Daan.dev

	OoohBoi Steroids for Elementor
		Version: 2.1.6
		Author: OoohBoi

	Photonic Gallery &amp; Lightbox for Flickr, SmugMug, Google Photos &amp; Others
		Version: 3.10
		Author: Sayontan Sinha

	Polylang
		Version: 3.6.1
		Author: WP SYNTEX

	Post SMTP
		Version: 2.9.2
		Author: Post SMTP

	Rank Math SEO with AI Best SEO Tools
		Version: 1.0.219
		Author: Rank Math

	Really Simple SSL
		Version: 8.1.3
		Author: Really Simple Plugins

	Relevanssi
		Version: 4.22.2
		Author: Mikko Saari

	ShortPixel Image Optimizer
		Version: 5.6.1
		Author: ShortPixel - Convert WebP/AVIF & Optimize Images

	Smash Balloon Instagram Feed
		Version: 6.3.1
		Author: Smash Balloon

	TablePress
		Version: 2.3.1
		Author: Tobias Bäthge

	Ultimate Addons for Elementor
		Version: 1.36.31
		Author: Brainstorm Force

	Widget Context
		Version: 1.3.2
		Author: Kaspars Dambis

	Wordfence Security
		Version: 7.11.5
		Author: Wordfence

	WPCode Lite
		Version: 2.1.12
		Author: WPCode

	WPS Hide Login
		Version: 1.9.15.2
		Author: WPServeur, NicolasKulka, wpformation

	WP Store Locator
		Version: 2.2.241
		Author: Tijmen Smit


== Must-Use Plugins ==
	burst_rest_api_optimizer.php
		Version: 
		Author: 

	vaa-cs.php
		Version: 
		Author: 


== Funktioner ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrationer ==
	


== Elementor-experiment ==
	Förbättrad inläsning av filer: Aktiv som standard
	Förbättrad CSS-laddning: Aktiv
	Inline Font Icons: Inaktiv
	Ytterligare anpassade brytpunkter: Inaktiv
	admin_menu_rearrangement: Inaktiv som standard
	Flexbox Container: Aktiv
	Uppgradera Swiper-bibliotek: Aktiv
	Behållare med rutnät: Aktiv
	Nested Elements Performance: Inaktiv som standard
	Optimized Control Loading: Aktiv
	Default to New Theme Builder: Aktiv som standard
	Elementor Home Screen: Aktiv som standard
	Topplist för redigerare: Aktiv
	Bygg med AI: Inaktiv
	Landningssidor: Inaktiv
	Nested Elements: Aktiv
	Uppskjuten inläsning av bakgrundsbilder: Aktiv
	Sidopanel: Inaktiv som standard
	Visa villkor: Aktiv
	Formulärinlämningar: Inaktiv
	Meny: Inaktiv som standard
	Taxonomifilter: Inaktiv


== Logg ==
	
Log: showing 20 of 272024-05-03 11:44:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-05-03 11:44:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.1',
  'to' => '3.21.2',
)]
2024-05-03 11:44:20 [info] elementor::elementor_updater Started 
2024-05-03 11:44:20 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-03 11:44:20 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-03 11:44:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-05-08 13:24:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.4',
  'to' => '3.21.5',
)]
2024-05-08 13:24:47 [info] elementor::elementor_updater Started 
2024-05-08 13:24:47 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-08 13:24:47 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-08 13:24:47 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.4',
  'to' => '3.21.5',
)]
2024-05-21 17:06:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.5',
  'to' => '3.21.6',
)]
2024-05-21 17:06:28 [info] elementor::elementor_updater Started 
2024-05-21 17:06:28 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-21 17:06:28 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-21 17:06:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.5',
  'to' => '3.21.6',
)]
2024-05-21 17:06:38 [info] elementor-pro::elementor_pro_updater Started 
2024-05-21 17:06:38 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-05-21 17:06:38 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-05-21 17:06:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.2',
  'to' => '3.21.3',
)]

PHP: showing 9 of 9PHP: 2024-05-02 08:08:59 [warning X 1302][../wp-content/plugins/elementor/includes/widgets/video.php::1076] Undefined array key &quot;aspect_ratio&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-02 10:24:22 [warning X 1348][../wp-content/plugins/elementor/includes/base/controls-stack.php::685] Undefined array key -1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-03 11:26:38 [warning X 30][../wp-content/plugins/elementor/includes/template-library/sources/local.php::1065] Attempt to read property &quot;post_type&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-04 07:11:05 [warning X 39][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-05-10 07:11:32 [warning X 1139][../wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key &quot;background_color_b&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-10 07:55:46 [warning X 2][../wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/fonts-manager.php::252] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-17 12:43:12 [warning X 18][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-05-17 15:23:44 [warning X 16][../wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::64] Undefined array key &quot;post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-05-17 19:40:20 [warning X 3][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 6 of 6JS: 2024-05-08 08:15:42 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.21.4:3:54583] Cannot read properties of undefined (reading &#039;activate&#039;) 
JS: 2024-05-09 05:34:14 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.21.5:3:703312] this.model.isValidChild is not a function 
JS: 2024-05-09 13:52:33 [error X 65][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.21.5:3:646902] Cannot read properties of undefined (reading &#039;model&#039;) 
JS: 2024-05-10 16:19:07 [error X 1][../wp-content/plugins/elementor-pro/assets/js/gallery.8ca9a354ce039d1ba641.bundle.min.js:2:5674] Cannot read properties of undefined (reading &#039;setSettings&#039;) 
JS: 2024-05-13 11:04:58 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.21.5:3:914069] Cannot read properties of undefined (reading &#039;val&#039;) 
JS: 2024-05-20 05:04:07 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.21.5:3:153456] Cannot read properties of null (reading &#039;scrollTo&#039;) 



== Elementor - Compatibility Tag ==
	
	Connect Polylang for Elementor: Kompatibilitet inte specificerad
	Elementor Pro: Kompatibilitet inte specificerad
	OoohBoi Steroids for Elementor: Kompatibilitet inte specificerad
	Ultimate Addons for Elementor: Kompatibilitet inte specificerad

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by hdd064** [comment]
I have exactly the same experience with the accordion

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved with the release of Elementor v3.23.0! 🥳 

The time to interaction should improve by 60%-98%, and performance overall should be better. For a full breakdown please read [this article](https://developers.elementor.com/elementor-3-23-developers-update/#:~:text=Performant%20Nested%20Elements%20In%20the%20Editor).

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
